Development/src/pages/devices/ChannelMappingMatrix.js

```diff
@@ -0,0 +1,514 @@
+import { useState } from 'react';
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableRow,
+    Tooltip,
+    Typography,
+} from '@material-ui/core';
+import {
+    ReferenceField,
+    ReferenceManyField,
+    SingleFieldList,
+} from 'react-admin';
+import get from 'lodash/get';
+import ChipConditionalLabel from '../../components/ChipConditionalLabel';
+import MappingButton from '../../components/MappingButton';
+import CollapseButton from '../../components/CollapseButton';
+
+const get_output_caps = (output_item, io) => {
+    return (
+        <>
+            <Typography variant="body1">{'Description:'}</Typography>
+            {output_item.properties.description}
+            <Typography variant="body1">{'Routable inputs:'}</Typography>
+            {output_item.caps.routable_inputs !== null
+                ? output_item.caps.routable_inputs
+                      .map(input_name => {
```

Comment:
How about `input_id` for this to distinguish the ID from the human - readable `properties.name`?

Reply:
I think there are a few other places where `name` (and `nmos_name`) is used to mean the input or output ID in the *io* object. Could we make all of these use `id` consistently?